umake/ui/cli/__init__.py

```diff
@@ -157,6 +158,65 @@ def mangle_args_for_default_framework(args):
     return result_args
 
 
+def get_frameworks_list(args):
+    """
+    Get a frameworks list based on the arguments.
+    Multiple forms of the frameworks list can ge given:
+        - List with all frameworks
+        - List with just only installed frameworks
+        - List with just installable frameworks
+
+    Args:
```

Comment:
I wouldn't really detail the args here, the function is short enough so that someone can understand it. You can remove the stenza then!